GoogleSignInSwift/Sources/GIDSignInButton_SwiftUI.swift

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import GoogleSignIn
+import CoreGraphics
+
+// MARK: - Custom Width
+@available(iOS 13.0, macOS 10.15, *)
+fileprivate struct Width {
+  let min, max: CGFloat
+}
+
+extension GIDSignIn {
+
+}
+
+// MARK: - Width and Text By Button Style
+@available(iOS 13.0, macOS 10.15, *)
+extension GIDSignInButtonStyle {
+  fileprivate var width: Width {
+    switch self {
+    case .iconOnly: return Width(min: CGFloat(kIconWidth), max: CGFloat(kIconWidth))
+    case .standard: return Width(min: 90, max: .infinity)
+    case .wide: return Width(min: 170, max: .infinity)
+    default:
+      fatalError("Unrecognized case for `GIDSignInButtonStyle: \(self)")
+    }
+  }
+
+  fileprivate var buttonText: String {
+    switch self {
+    case .wide: return "Sign in with Google"
+    case .standard: return "Sign in"
+    case .iconOnly: return ""
+    default:
+      fatalError("Unrecognized case for `GIDSignInButtonStyle: \(self)")
+    }
+  }
+}
+
+// MARK: - Button Style
+@available(iOS 13.0, macOS 10.15, *)
+struct SwiftUIButtonStyle: ButtonStyle {
+  let style: GIDSignInButtonStyle
+
+  func makeBody(configuration: Configuration) -> some View {
+    configuration.label
+      .frame(minWidth: style.width.min,
+             maxWidth: style.width.max,
+             minHeight: CGFloat(kButtonHeight),
+             maxHeight: CGFloat(kButtonHeight))
+      .foregroundColor(.blue)
```

Comment:
The default text color for the SwiftUI button seems to be blue rather than grey as in GIDSignInButton, is this the cause?